### PR TITLE
[typespec-next] React to azure/typespec-azure#3177

### PR DESCRIPTION
**Pull request by mikeharder**
- Azure Core Rule Omni-PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "@azure-tools/cadl-azure-resource-manager": "0.26.0",
     "@azure-tools/cadl-providerhub": "0.26.0",
     "@azure-tools/typespec-autorest": "next",
-    "@azure-tools/typespec-azure-core": "next",
+    "@azure-tools/typespec-azure-core": "0.32.0-dev.6",
```
Comment (mikeharder) [inline package.json:10]: This is temporary and will be unpinned in the next PR to `typespec-next`

File: package.json [hunk 2]
```diff
@@ -6,17 +6,17 @@
...
-    "@azure-tools/typespec-azure-resource-manager": "next",
-    "@azure-tools/typespec-client-generator-core": "next",
-    "@azure-tools/typespec-providerhub": "next",
-    "@typespec/compiler": "next",
-    "@typespec/http": "next",
-    "@typespec/openapi": "next",
-    "@typespec/rest": "next",
-    "@typespec/versioning": "next",
-    "@typespec/lint": "next",
+    "@azure-tools/typespec-autorest": ">=0.32.0-dev",
```
Comment (mikeharder) [inline package.json:9]: Temporary until https://github.com/Azure/typespec-azure/pull/3238 is merged

File: eng/pipelines/typespec-ci.yml [hunk 3]
```diff
@@ -38,8 +38,8 @@ jobs:
   - script: rm package-lock.json
     displayName: rm package-lock.json
 
-  - script: npm install
-    displayName: npm install
+  - script: npm install --force
```
Comment (mikeharder) [inline eng/pipelines/typespec-ci.yml:41]: Temporary until https://github.com/Azure/typespec-azure/pull/3238 is merged

File: eng/pipelines/typespec-ci.yml [hunk 4]
```diff
@@ -38,8 +38,8 @@ jobs:
...
+    displayName: npm install --force
```
Comment (mikeharder) [inline eng/pipelines/typespec-ci.yml:42]: Temporary until https://github.com/Azure/typespec-azure/pull/3238 is merged

File: package.json [hunk 5]
```diff
@@ -6,17 +6,17 @@
...
+    "@azure-tools/typespec-azure-resource-manager": "https://artprodcus3.artifacts.visualstudio.com/A0fb41ef4-5012-48a9-bf39-4ee3de03ee35/29ec6040-b234-4e31-b139-33dc4287b756/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2F6dXJlLXNkay9wcm9qZWN0SWQvMjllYzYwNDAtYjIzNC00ZTMxLWIxMzktMzNkYzQyODdiNzU2L2J1aWxkSWQvMjg4MTQ2MC9hcnRpZmFjdE5hbWUvcGFja2FnZXM1/content?format=file&subPath=%2Fazure-tools-typespec-azure-resource-manager-0.31.0.tgz",
+    "@azure-tools/typespec-client-generator-core": ">=0.32.0-dev",
+    "@azure-tools/typespec-providerhub": ">=0.32.0-dev",
+    "@typespec/compiler": ">=0.46.0-dev",
+    "@typespec/http": ">=0.46.0-dev",
+    "@typespec/openapi": ">=0.46.0-dev",
+    "@typespec/rest": ">=0.46.0-dev",
+    "@typespec/versioning": ">=0.46.0-dev",
+    "@typespec/lint": ">=0.46.0-dev",
```
Comment (mikeharder) [inline package.json:19]: Temporary until https://github.com/Azure/typespec-azure/pull/3238 is merged

File: specification/contosowidgetmanager/Contoso.WidgetManager/main.tsp [hunk 6]
```diff
@@ -9,6 +9,7 @@ using TypeSpec.Rest;
 using TypeSpec.Versioning;
 using Azure.Core;
 
+#suppress "@azure-tools/typespec-azure-core/auth-required" "MUST fix in next update"
```
Comment (tjprescott) [inline specification/contosowidgetmanager/Contoso.WidgetManager/main.tsp:12]: Contoso.WidgetManager is literally just an example, so this one I think we should just fix. 
```suggestion
@useAuth(AadOauth2Auth<["https://contoso.azure.com/.default"]>)
```

**Review by tjprescott — CHANGES_REQUESTED** [review]

File: specification/contosowidgetmanager/Contoso.WidgetManager/main.tsp (hunk 6 above)
Comment (timotheeguerin) [reply specification/contosowidgetmanager/Contoso.WidgetManager/main.tsp:12]: +1 on that. If this is actually just an example we should try to not show suppression

Comment (mikeharder) [reply specification/contosowidgetmanager/Contoso.WidgetManager/main.tsp]: The only remaining suppression in this example is temporary and required until https://github.com/Azure/typespec-azure/issues/3107 is fixed.

**Comment by mikeharder** [comment]
@tjprescott: For the other two violations, do you think we should suppress or work with the spec owners to try to fix?

File: specification/contosowidgetmanager/data-plane/Azure.Contoso.WidgetManager/preview/2022-11-01-preview/widgets.json [hunk 7]
```diff
@@ -289,7 +308,7 @@
             "description": "The request has succeeded.",
             "schema": {
               "type": "object",
-              "description": "Provides the status of a resource operation.",
```
Comment (mikeharder) [inline specification/contosowidgetmanager/data-plane/Azure.Contoso.WidgetManager/preview/2022-11-01-preview/widgets.json:292]: Reverting temporary change needed in last PR, to use one PR per reaction.

**Pull request merged**